operator-common/src/test/java/io/strimzi/operator/common/operator/resource/AbstractResourceOperatorIT.java

```diff
@@ -105,6 +106,7 @@ public void testCreateModifyDelete(VertxTestContext context)    {
             }))
             .compose(rr -> op.reconcile(namespace, RESOURCE_NAME, null))
             .setHandler(context.succeeding(rrDeleted -> {
+                TestUtils.waitFor("resouce deleted", 1_000, 30_000, () -> op.get(namespace, RESOURCE_NAME) == null);
```

Comment:
Why do we have to wait here now and did not had to wait before? That sounds like it will break whatever place is using it and is expecting the old behavior.

Reply:
It is flaky now but I am afraid it is the same as https://github.com/strimzi/strimzi-kafka-operator/pull/1424/files#diff-210c63328d6cf1d8f3f414fdd5244bf8R129